controllers/users.js

```diff
@@ -0,0 +1,156 @@
+const jwt = require('jsonwebtoken');
+const passport = require('passport');
+const User = require('../models/user');
+
+require('dotenv').config();
+
+const secret = process.env.SECRET;
+
+const auth = (req, res, next) => {
+  passport.authenticate(
+    'jwt',
+    { session: false },
+    (err, user) => {
+      if (!user || err) {
```

Comment:
tu trzeba jeszcze sprawdzić czy user.token jest taki sam jak token w req (z przodu trzeba odjąć to "bareare " przy porównaniu). Teraz mogę się wylogować a token nadal sprawdza '/current'

Reply:
done